numpy/core/src/umath/ufunc_object.c

```diff
@@ -3842,10 +3842,13 @@ PyUFunc_GenericReduction(PyUFuncObject *ufunc, PyObject *args,
     PyArray_Descr *otype = NULL;
     PyArrayObject *out = NULL;
     int keepdims = 0;
-    static char *kwlist1[] = {"array", "axis", "dtype",
-                                "out", "keepdims", NULL};
-    static char *kwlist2[] = {"array", "indices", "axis",
-                                "dtype", "out", NULL};
+    static char *reduce_kwlist[] = {
+        "array",            "axis", "dtype", "out", "keepdims", NULL};
```

Comment:
You'd argue that that the alignment of argument names isn't worth the extra spacing then?